configure.ac

```diff
@@ -435,6 +434,21 @@ AM_CONDITIONAL([ENABLE_HTTPS], [test "${enable_https}" = "yes"])
 
 # -----------------------------------------------------------------------------
 # JSON-C
+
+if test "${enable_jsonc}" != "no" -a -z "${JSONC_LIBS}"; then
+	# Try and detect manual static build presence (from netdata-installer.h)
+	AC_MSG_CHECKING([if staticaly built json-c is present])
+	if test -f "externaldeps/jsonc/libjson-c.a"; then
+		AC_DEFINE([LINK_STATIC_JSONC], [1], [static json-c should be used])
```

Comment:
I think that we should do a test compile to check that it works. The symbol check that is already there for the system installed library would be appropriate.